### PR TITLE
Render image clean up, Add Amp param, Add initial image concept

### PR DESCRIPTION
**Pull request by PascalPiche**

### CODE REVIEW

File: workspace/utilities/lib/render-image.xsl [hunk 1]
```diff
@@ -450,11 +498,15 @@
...
-			<xsl:with-param name="f" select="name" />
+			<xsl:with-param name="b" select="image-credit" />
+			<xsl:with-param name="c" select="alt" />
+			<xsl:with-param name="d" select="titre" />
+			<xsl:with-param name="e" select="title" />
+			<xsl:with-param name="f" select="nom" />
+			<xsl:with-param name="g" select="name" />
+			<xsl:with-param name="h" select="legende" />
+			<xsl:with-param name="i" select="legend" />
+			<xsl:with-param name="j" select="caption" />
```
Comment (nitriques) [inline workspace/utilities/lib/render-image.xsl]: Please please please rewrite that shit with xml values...

**Comment by hotdoy** [comment]
I say merge this one for now. Default content template is another problem.

**Comment by nitriques** [comment]
>  Default content template is another problem. And I'll gladly rewrite this one and all the others throughout our template.

Can wait to see that @hotdoy

**Comment by nitriques** [comment]
@dominic-blain Are you ok with it ?

**Pull request merged**